lib/spring/client/binstub.rb

```diff
@@ -13,8 +13,9 @@ class Binstub < Command
       # should cause the "unsprung" version of the command to run.
       LOADER = <<CODE
 begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError
+  load spring_loader = File.expand_path('../spring', __FILE__)
```

Comment:
Yes, but it would also break the existing regex to check for the old/new style binstub.